src/assets/scripts/client/aircraft/Pilot/Pilot.js

```diff
@@ -648,7 +648,7 @@ export default class Pilot {
 
         const readback = {};
         readback.log = 'intercept glidepath';
-        readback.log = 'intercept glidepath';
+        readback.say = 'intercept glidepath';
 
         return [true, readback];
```

Comment:
```suggestion
        return [true, 'intercept glidepath'];
```

Reply:
Also removed the portion of code that creates an object.